gradle/codequality/HEADER.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```

Comment:
Should this be 2022-2023?

Reply:
We have been discussing this [here](https://github.com/google/dwh-migration-tools/pull/159#discussion_r1136933548). What's your thoughts?

Reply:
Changing to 2022-2023...